content/en/intelligent_test_runner/_index.md

```diff
@@ -65,6 +65,9 @@ Due to the [limitations](#out-of-the-box-configuration-limitations) described ab
 
 If there are other branches you want to exclude, add them on the Test Service Settings page. The query bar supports using the wildcard character `*` to exclude any branches that match, such as `release_*`.
 
+Excluded branches still collect per-test code coverage, so you might notice a performance impact on your excluded branches. However, code coverage is enabled
```

Comment:
Excluded branches collect per-test code coverage, which will have a performance impact on the total testing time. However, this performance impact is mitigated by only collecting code coverage when Datadog detects that running with code coverage will generate enough new coverage information that it offsets the cost of collecting the coverage.